bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckWorkItem.java

```diff
@@ -524,9 +524,11 @@ public Collection<WorkItem> prRun(ScratchArea scratchArea) {
                 } catch (IOException e) {
                     throw new UncheckedIOException(e);
                 }
-                var metadata = pr.repository().forge().search(hash);
-                if (metadata.isPresent()) {
-                    var message = CommitMessageParsers.v1.parse(metadata.get().message());
+                var forge = pr.repository().forge();
+                var repoName = forge.search(hash);
+                if (repoName.isPresent()) {
+                    var metadata = forge.repository(repoName.get()).flatMap(repository -> repository.commit(hash));
```

Comment:
I find the naming of this variable confusing. Can we just call it `commit`?

Reply:
Sure